.infrastructure/helm-chart/todoapp/templates/pv.yml

```diff
@@ -0,0 +1,14 @@
+apiVersion: v1
+kind: PersistentVolume
+metadata:
+  name: {{ .Chart.Name }}-pv
+  namespace: {{ .Values.todoapp.namespace }}
```

Comment:
PersistentVolumes are not namespaced resources. Remove the `namespace` field from the metadata to avoid deployment issues.